examples/usecases/transformers-next-item-prediction.ipynb

```diff
@@ -87,7 +87,7 @@
      "name": "stderr",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        ["checkin"]
</pre><p>why do we need <code>get_lib()</code> ?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/1045/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>I believe this is to not introduce a dependency on cudf (when running in an env without a GPU it won't be available). </p>